pandas/tests/io/formats/data/html/gh53885_expected_output.html

```diff
@@ -0,0 +1,26 @@
+<table border="1" class="dataframe">
```

Comment:
Could you just assign the data in this file to a string?

Reply:
sure @mroeschke 
Had mainly done it this way since other tests seemed to do the same.